### ISSUE TITLE
Streaming mp3 not working

### ISSUE THREAD

**Issue by 0xRumple** [martijn00/XamarinMediaManager#191]
Not sure if it's really broken.

I have written a question in stackoverflow to determine the issue:

http://stackoverflow.com/questions/43876185/streaming-mp3-in-xamarin-android-mediamanager-not-working

**Comment by jmartine2** [comment]
I just wanted to note that the sample app does not work for me either with the streaming audio. If I change the URL's hardcoded in the source code to file paths on my device, it works.

**Comment by 0xRumple** [comment]
@jmartine2 
Yeah... it seems broken.

Still looking for a way to stream audio !
It's really trivial, but I tried these libraries too with no luck:
1- exoplayer: broken too
2- ffmpeg (github.com/wseemann/FFmpegMediaPlayer): when I use "stop" function, app crashes (text relocation problem)
Any idea ?

**Comment by jmartine2** [comment]
It used to work for me, but not anymore. It may have stopped working when I switched to the ExoPlayer implementation. Have you tried the stock Android implementation? In the sample app, set ShouldUseExoPlayer to false.

**Comment by baptiste09** [comment]
There is an issue into ExoPlayerAudioService. In GetSource(), URLUtil.IsHttpUrl and URLUtil.IsHttpsUrl should be called using url as parameter, not escapedUrl.

**Comment by 0xRumple** [comment]
@jmartine2 
I have tried the stock android implementation like this:

`

	string url1 = @"http://www.jamesreams.com/wp-content/uploads/2013/01/Born-to-Roll-clip.mp3";

	string url2 = @"http://www.wav-sounds.com/answering_machine/rappin.wav";

	var mp = new MediaPlayer();
	mp.Prepared += (sender, args) =>
	{
		mp.Start();
	};
	mp.SetDataSource(url2);
	mp.PrepareAsync();`

Working fine with the wav url (url2)... not fine with the mp3 url (url1).
It gives (Chromium Error http 403) in the logcat.

Right now I will try to bind this one:
https://github.com/wseemann/ServeStream
If it doesn't work too, I will try this one:
https://github.com/USBhost/MX_FFmpeg

I will be back :)

**Comment by kizzonia** [comment]
AM having same issue

**Comment by dskow** [comment]
In VS 2017, I created a "Cross Platform App (Xamarin)" project and  added Plugin.MediaManager.Forms 0.4.0 and Plugin.MediaManager 0.4.0 nuget packages.

I add the following line to my code.

await CrossMediaManager.Current.Play("http://www.montemagno.com/sample.mp3");

It works by default in the UWP phone emulator on Hyper-v. However, I ran into the "no audio playing" problem using Android Emulator on Hyper-v. It turns out that the network is set to internal. So the http://www.montemagno.com/sample.mp3 could not be found. My workaround steps to have the app play sound:

    Hyper-v -> Virtual Switch Manager, add an external network.
    Hyper-v -> Virtual Machines->Settings, add new hardware->Network adapter and set to external network.
    "Visual Studio Emulator for Android" desktop app, launch phone vm,
    in Visual Studio, deploy and run app.

App sounds should now work from external source.

The android phone emulator VS 2017 options are Debug/Any CPU/App.Android/5.7" Marshmallow (6.0.0) XHDPI Phone (Android 6.0 - API 23)
The UWP phone emulator VS 2017 options are Debug/Any CPU/App.UWP(Universal Windows)/Mobile Emulator 10.0.15063.0 WVGA 4 inch 512MB
